examples/demo/src/i18n/en.ts

```diff
@@ -40,6 +40,7 @@ const customEnglishMessages: TranslationMessages = {
     resources: {
         customers: {
             name: 'Customer |||| Customers',
+            forcedCaseName: 'Customer |||| Customers',
```

Comment:
I don't understand why you add these translations, they should not be necessary.